docs/guides/monitoring/README.md

```diff
@@ -98,8 +98,8 @@ Then you should have a packaging directory that includes all the files above:
 
 ```yaml
 package:
-  include:
-    - src/*.js
+  patterns:
+    - '!src/*.js'
```

Comment:
This is a readme example that use to do:
```yaml
package:
  include:
    - src/*.js
    - dist/*.js
    - dist/*.js.map
```

but I think that's a better example to exclude sources if we add the compiled files with their source-maps from `dist` and exclude the original files from `src`.

Do you dissagree?

Reply:
@juanjoDiaz this example intentionally includes sources in a bundle (note the sentence: _To use a sourcemap, ensure that your packaging directory includes the compiled source_)

Weather it's good example, and whether it could be improved it's the matter that's out of the scope of this PR. We may eventually open another issue to discuss it.